Shaders/MeshChainAdd.shader

```diff
@@ -53,7 +53,7 @@ Shader "VRLineRenderer/MeshChain - Additive"
             // prevent overlapping lines from adding too much color,
             // we set the alpha value to one after visiting a pixel.
             Blend OneMinusDstAlpha One, One One
-            BlendOp Add
+            BlendOp Add, Max

```

Comment:
Do these fix the unclamped alpha channel issues we're seeing in the custom EVR build? I haven't gone through and refactored EVR to use this yet, but I can try just this change in the current version and see